applications/nrf5340_audio/dfu/conf/Kconfig.dfu

```diff
@@ -111,6 +111,18 @@ config MCUBOOT_IMAGE_VERSION
 	string
 	default "0.0.0+0"
 
+config LOG_DFU_ENTRY_LEVEL
+	int "Log level for dfu entry module"
```

Comment:
"dfu" should be capitalized

Reply:
Change to "Log level for DFU entry module" .